docs/core/install/linux-snap.md

```diff
@@ -15,6 +15,9 @@ A snap is a bundle of an app and its dependencies that works without modificatio
 > [!CAUTION]
 > Snap packages aren't supported in WSL2 on Windows 10. As an alternative, use the [`dotnet-install` script](linux-scripted-manual.md#scripted-install) or the package manager for the particular WSL2 distribution. It's not recommended but you can try to enable snap with an [unsupported workaround from the snapcraft forums](https://forum.snapcraft.io/t/running-snaps-on-wsl2-insiders-only-for-now/13033).
 
+> [!CAUTION]
+> Snap installations of .NET may have problems running [.NET tools](../tools/global-tools.md). If you wish to use .NET tools, we recommend that you install .NET using the [`dotnet-install` script](linux-scripted-manual.md#scripted-install) or the package manager for the particular Linux distribution.
```

Comment:
Would it be worth it to call out that even `dotnet watch` is broken?

Reply:
probably, can you suggest an edit?

Reply:
I modified your file, clarifying a little bit more.

See these two commits:
- [commit 1](https://github.com/dotnet/docs/pull/31045/commits/25635f0018662093dea858e9e88475e4081ed3ae)
- [commit 2](https://github.com/dotnet/docs/pull/31045/commits/6ab46bdc20dbcf9f370359d1673b1f4194c86034)